infra/events/src/main/java/com/simprints/infra/events/event/cosync/CoSyncEvents.kt

```diff
@@ -1,9 +1,9 @@
-package com.simprints.feature.clientapi.models
+package com.simprints.infra.events.event.cosync
 
 import androidx.annotation.Keep
 import com.simprints.infra.events.event.domain.models.Event
 
 @Keep
-internal data class CoSyncEvents(
+data class CoSyncEvents(
```

Comment:
Is the removal of the `internal` keyword intentional?

Reply:
Same for this one.